src/components/pages/AppOverview/AddRolesOverlay.tsx

```diff
@@ -0,0 +1,303 @@
+
+            setRolesPreviews(roles?.splice(1))
+            setRolesDescription(roleDescription?.splice(1))
+
+            setUploadCSVError(false)
+          } else {
+            setRolesPreviews([])
+            setRolesDescription([])
+
+            setUploadCSVError(true)
+          }
+        }
+        reader.readAsText(file)
+      })
+  }
+  const tableData: TableType = {
+    head: [t('content.addRoles.newRolesToAdd')],
+    body: rolesPreviews?.map((item) => [item]),
+  }
+
+  const handleConfirm = async () => {
+    setIsLoading(true)
+
+    const rolesDescriptionData = rolesPreviews.map((data, i) => [
+      data,
+      rolesDescription[i],
+    ])
+
+    const updatedData = {
+      appId: appId,
+      body: rolesDescriptionData?.map((item) => ({
+        role: item[0],
+        descriptions: [
+          {
+            languageCode: 'en',
+            description: item[1],
+          },
+          {
+            languageCode: 'de',
+            description: item[1],
+          },
+        ],
+      })),
+    }
+
+    await updateActiveApp(updatedData)
+      .unwrap()
+      .then(() => {
+        navigate(`/${PAGES.APPOVERVIEW}`, {
+          state: 'add-roles-success',
+        })
+        success(t('content.addRoles.successMsg'))
+      })
+      .catch((err) => {
+        setIsLoading(false)
+        error(t('content.addRoles.errorMsg'), '', err)
+      })
+  }
+
+  return (
+    <div className="detailsOverlay">
+      <Dialog
+        open={openDialog}
+        additionalModalRootStyles={{
+          width: '60%',
+        }}
+      >
+        <DialogHeader
+          title={t('content.addRoles.uploadAdditionalRoles')}
+          intro={t('content.addRoles.uploadAdditionalRolesDescription')}
+          closeWithIcon={true}
+          onCloseWithIcon={() => {
+            handleOverlayClose()
+          }}
+        />
+        <DialogContent
+          sx={{
+            padding: '0px 120px 40px 120px',
+          }}
+        >
+          <Grid
+            item
+            xs={12}
+            sx={{ mr: 2, mt: '5px', mb: 5, textAlign: 'center' }}
+          >
+            <a
+              href="../../app-provider-role-upload-example.csv"
+              style={{ textDecoration: 'none' }}
+              download
+            >
+              <Button
+                variant="outlined"
+                endIcon={<FileDownloadOutlinedIcon />}
+                sx={{ fontSize: '16px' }}
+                size="small"
+              >
+                {t('content.apprelease.technicalIntegration.template')}
+              </Button>
+            </a>
+            <Button
+              sx={{ ml: 2, fontSize: '16px' }}
+              color="secondary"
+              size="small"
+              variant="contained"
+              onClick={() =>
+                window.open(
+                  'https://portal.dev.demo.catena-x.net/documentation/?path=docs%2F04.+App%28s%29%2FRelease-Process%2FApp+Release+Workflow.md',
```

Comment:
please make this a relative link by removing this part:

`https://portal.dev.demo.catena-x.net`